.gitignore

```diff
@@ -5,8 +5,11 @@
 **/dist/
 **/CMakeFiles/
 **/download/
+**/astyle/
```

Comment:
This really only exists in your private (?) branch.
```suggestion
```

Reply:
Because we had astyle as submodule, I thought it will be a good idea to add this to the gitignore, so the submodule cannot be reborn.